pkg/docker/docker.go

```diff
@@ -739,6 +740,23 @@ func getImageName(name string) string {
 	return name
 }
 
+func GetFullImageName(imageName, serverAddress string) (fullImageName string) {
```

Comment:
put this function to package `api` or `run`。Not here

Reply:
ok, I put it to `api/types.go`.